### PR TITLE
Fix `video_simple` snippet

### PR DESCRIPTION
**Pull request by jprochazk**
### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7313?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7313?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/7313)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: docs/snippets/snippets.toml [hunk 1]
```diff
@@ -187,3 +187,4 @@ quick_start = [ # These examples don't have exactly the same implementation.
 [extra_args]
 "archetypes/asset3d_simple" = ["$config_dir/../assets/cube.glb"]
 "archetypes/asset3d_out_of_tree" = ["$config_dir/../assets/cube.glb"]
+"archetypes/video_simple" = ["$config_dir/../assets/cube.glb"]
```
Comment (teh-cmc) [inline docs/snippets/snippets.toml:190]: that looks fishy

Comment (jprochazk) [reply docs/snippets/snippets.toml]: oops

File: docs/snippets/all/archetypes/video_simple.cpp [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Log a simple 3D asset.
```
Comment (teh-cmc) [inline docs/snippets/all/archetypes/video_simple.cpp:1]: image

File: docs/snippets/all/archetypes/video_simple.cpp [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+#include <rerun.hpp>
+
+#include <filesystem>
+#include <iostream>
+#include <string>
+
+int main(int argc, char* argv[]) {
+    if (argc < 2) {
+        std::cerr << "Usage: " << argv[0] << " <path_to_asset.[gltf|glb|obj|stl]>" << std::endl;
```
Comment (teh-cmc) [inline docs/snippets/all/archetypes/video_simple.cpp:11]: ditto

File: docs/snippets/all/archetypes/video_simple.rs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+//! Log a simple 3D asset.
```
Comment (teh-cmc) [inline docs/snippets/all/archetypes/video_simple.rs:1]: ditto

File: docs/snippets/all/archetypes/video_simple.rs [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+use rerun::external::anyhow;
+
+fn main() -> anyhow::Result<()> {
+    let args = std::env::args().collect::<Vec<_>>();
+    let Some(path) = args.get(1) else {
+        anyhow::bail!("Usage: {} <path_to_asset.[gltf|glb|obj|stl]>", args[0]);
```
Comment (teh-cmc) [inline docs/snippets/all/archetypes/video_simple.rs:8]: ditto

**Review by teh-cmc — APPROVED** [review]

**Pull request merged**